python/CMakeLists.txt

```diff
@@ -391,7 +391,8 @@ set(CYTHON_EXTENSIONS
     _compute
     _csv
     _feather
-    _json)
+    _json
+    _hdfsio)
```

Comment:
Nit, but can you keep alphabetical order?

Reply:
Sorted alphabetically. Also there was `_fs` that was before `_compute` and thus moved that one too.